src/components/RankingItem.tsx

```diff
@@ -0,0 +1,81 @@
+/** @jsxImportSource @emotion/react */
+import styled from '@emotion/styled';
+import type { Product } from '@/types/Product';
+
+interface Props {
+  item: Product;
+  rank: number;
+}
+
+const Card = styled.li`
+  display: flex;
+  flex-direction: column;
+`;
+
+const ItemImage = styled.div`
+  position: relative;
+  border-radius: 4px;
+  overflow: hidden;
+
+  img {
+    width: 100%;
```

Comment:
직접 실행해보면 이미지에 `border-radius`가 제대로 적용되지 않았는데요! 아래처럼 수정해주셔야 할 것 같아요.

```suggestion
  overflow: hidden;

  img {
    border-radius: 4px;
    width: 100%;
```